### ISSUE TITLE
[Bug] Platform asusrouter does not generate unique IDs

### ISSUE THREAD

**Issue by mihsu81** [Vaskivskyi/ha-asusrouter#528]
### The problem

After updating to 0.19.0 I can see errors in HA about asusrouter not generating unique IDs.

```
Logger: homeassistant.components.switch
Source: helpers/entity_platform.py:540
Integration: Switch (documentation, issues)
First occurred: 23:27:19 (55 occurrences)
Last logged: 23:27:23

Platform asusrouter does not generate unique IDs. ID asusrouter_3c:7c:3f:da:ed:58_guest_2_4_ghz_1 already exists - ignoring switch.router_guest_2_4_ghz_1
Platform asusrouter does not generate unique IDs. ID asusrouter_3c:7c:3f:da:ed:58_parental_control already exists - ignoring switch.router_parental_control
Platform asusrouter does not generate unique IDs. ID asusrouter_3c:7c:3f:da:ed:58_port_forwarding already exists - ignoring switch.router_port_forwarding
Platform asusrouter does not generate unique IDs. ID asusrouter_3c:7c:3f:da:ed:58_open_vpn_client_1 already exists - ignoring switch.router_openvpn_client_1
Platform asusrouter does not generate unique IDs. ID asusrouter_38:2c:4a:aa:5f:60_wireless_5_ghz already exists - ignoring switch.living_room_bridge_wireless_5_ghz
```
```
Logger: homeassistant.components.sensor
Source: helpers/entity_platform.py:540
Integration: Sensor (documentation, issues)
First occurred: 23:27:14 (149 occurrences)
Last logged: 23:27:23

Platform asusrouter does not generate unique IDs. ID asusrouter_38:2c:4a:aa:5f:60_bridge_upload_speed already exists - ignoring sensor.living_room_bridge_bridge_upload_speed
Platform asusrouter does not generate unique IDs. ID asusrouter_38:2c:4a:aa:5f:60_wired_download already exists - ignoring sensor.living_room_bridge_wired_download
Platform asusrouter does not generate unique IDs. ID asusrouter_38:2c:4a:aa:5f:60_wired_download_speed already exists - ignoring sensor.living_room_bridge_wired_download_speed
Platform asusrouter does not generate unique IDs. ID asusrouter_38:2c:4a:aa:5f:60_wired_upload already exists - ignoring sensor.living_room_bridge_wired_upload
Platform asusrouter does not generate unique IDs. ID asusrouter_38:2c:4a:aa:5f:60_wired_upload_speed already exists - ignoring sensor.living_room_bridge_wired_upload_speed
```

### Your device model

RT-AX88U

### Firmware type

Merlin

### Firmware version

3.0.0.4.388.1

### Integration version with the issue

0.19.0

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

2023.2.2

### Diagnostics information

[config_entry-asusrouter-abc26babac5f76cbcf9c7deb0f2508e3.json.txt](https://github.com/Vaskivskyi/ha-asusrouter/files/10679630/config_entry-asusrouter-abc26babac5f76cbcf9c7deb0f2508e3.json.txt)
[config_entry-asusrouter-eea7c56fddb0fb9fba8c722811ae8749.json.txt](https://github.com/Vaskivskyi/ha-asusrouter/files/10679631/config_entry-asusrouter-eea7c56fddb0fb9fba8c722811ae8749.json.txt)

**Comment by Vaskivskyi** [comment]
Hello. Does it show on any HA reboot or only when you are manually reloading AsusRouter?

**Comment by mihsu81** [comment]
The errors appear in both cases.

Noticed another error now.
```
Logger: homeassistant.components.button
Source: helpers/entity_platform.py:540
Integration: Button (documentation, issues)
First occurred: 12:43:28 (2 occurrences)
Last logged: 12:43:38

Platform asusrouter does not generate unique IDs. ID asusrouter_3c:7c:3f:da:ed:58_restart_firewall already exists - ignoring button.router_restart_firewall
Platform asusrouter does not generate unique IDs. ID asusrouter_38:2c:4a:aa:5f:60_restart_firewall already exists - ignoring button.living_room_bridge_restart_firewall
```

**Comment by smarthomefamilyverrips** [comment]
@Vaskivskyi same here:

Logger: homeassistant.components.button
Source: helpers/entity_platform.py:540
Integration: Button (documentation, issues)
First occurred: February 18, 2023 at 19:34:53 (1 occurrences)
Last logged: February 18, 2023 at 19:34:53

Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_restart_firewall already exists - ignoring button.xt12_restart_firewall

AND

Logger: homeassistant.components.sensor
Source: helpers/entity_platform.py:540
Integration: Sensor (documentation, issues)
First occurred: February 18, 2023 at 19:34:53 (16 occurrences)
Last logged: February 18, 2023 at 19:34:53

Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_5_ghz_download already exists - ignoring sensor.xt12_5_ghz_download
Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_5_ghz_download_speed already exists - ignoring sensor.xt12_5_ghz_download_speed
Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_5_ghz_upload already exists - ignoring sensor.xt12_5_ghz_upload
Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_5_ghz_upload_speed already exists - ignoring sensor.xt12_5_ghz_upload_speed
Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_wan_download already exists - ignoring sensor.xt12_wan_download

AND

Logger: homeassistant.components.switch
Source: helpers/entity_platform.py:540
Integration: Switch (documentation, issues)
First occurred: February 18, 2023 at 19:34:53 (19 occurrences)
Last logged: February 18, 2023 at 19:34:53

Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_guest_2_4_ghz_1 already exists - ignoring switch.xt12_guest_2_4_ghz_1
Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_parental_control already exists - ignoring switch.xt12_parental_control
Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_port_forwarding already exists - ignoring switch.xt12_port_forwarding
Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_open_vpn_client_1 already exists - ignoring switch.xt12_openvpn_client_1
Platform asusrouter does not generate unique IDs. ID asusrouter_04:42:1a:c7:51:b0_wireless_2_4_ghz already exists - ignoring switch.xt12_wireless_2_4_ghz

**Comment by nickaggie** [comment]
+1 for this issue. I am experiencing the same thing with my RT-AX88U on Merlin 3.0.0.4.388.1

**Comment by Vaskivskyi** [comment]
Hello. Just letting you know, that the issue and potential fix are still in progress

**Comment by smarthomefamilyverrips** [comment]
👍🏻👍🏻👍🏻

**Comment by Vaskivskyi** [comment]
I am happy to announce, that this issue will be fixed in the next release

**Comment by smarthomefamilyverrips** [comment]
> I am happy to announce, that this issue will be fixed in the next release

👍🏻👍🏻👍🏻

**Comment by Vaskivskyi** [comment]
Hello. The issue should be fixed in 0.22.0, which is already available via HACS

**Comment by smarthomefamilyverrips** [comment]
@Vaskivskyi Thanks 👍🏻
